lib/turbo_tests/runner.rb

```diff
@@ -2,13 +2,21 @@
 
 require "json"
 require "parallel_tests/rspec/runner"
+require "parallel_tests/tasks"
 
 require_relative "../utils/hash_extension"
 
 module TurboTests
   class Runner
     using CoreExtensions
 
+    def self.create(count)
+      ENV["PARALLEL_TEST_FIRST_IS_1"] = "true"
+      command = ["bundle", "exec", "rake", "db:create", "RAILS_ENV=#{ParallelTests::Tasks.rails_env}"]
+      args = { count: count.to_s }
+      ParallelTests::Tasks.run_in_parallel(command, args)
+    end
```

Comment:
I'm not sure if it should be added to the `Runner`.

I understand it'd be nicer to reduce the number of steps on the user side. But in this case, it seems, that wrapping two commands in a bash/rake command fits better.